scripts/convert_celeba_to_numpy.py

```diff
@@ -0,0 +1,28 @@
+"""
+A simple script to convert the entire `CelebA` dataset to a numpy array for easier handling by the `pythae` library.
+"""
+
+import pathlib
+
+import torchvision
+import numpy as np
+
+from latte.dataset import celeba
+
+
+def main():
+    _ROOT = pathlib.Path("/cluster/work/boeva/asvete/dataset/raw")
```

Comment:
Not at this PR, but we should make it generic, I think

Reply:
Damn, yes, didn't catch this...